demos/dotnet/dotnet/dotnet.cs

```diff
@@ -27,7 +28,10 @@ static void Main(string[] args)
             Log.IsEnabled = true;
         }
 
-        switch (args[0])
+        var commandArgument = args[0];
```

Comment:
Agreed. Let's discuss the behavior of the command parsing logic to make it more flexible and scalable.